src/Runner.Worker/Container/ContainerHooks/HookContainer.cs

```diff
@@ -0,0 +1,64 @@
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+
+namespace GitHub.Runner.Worker.Container.ContainerHooks
+{
+    public class HookContainer : HookArgs
+    {
+        public string Id { get; set; }
+        public string DisplayName { get; set; }
```

Comment:
A lot of these fields aren't in the [ADR](https://github.com/actions/runner/blob/thboop/ContainerHookADR/docs/adrs/1891-container-hooks.md), so we don't plan on using them in our hook examples. 

I think we should remove them from this. We can always add them back later if we need them, but its very hard to remove them later.

Reply:
Will work on syncing these 👍